collectorsapp_2019-08-05.sql

```diff
@@ -0,0 +1,61 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 192.168.20.20 (MySQL 5.6.44)
+# Database: collectorsapp
+# Generation Time: 2019-08-05 13:22:34 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table grocery_item
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `grocery_item`;
+
+CREATE TABLE `grocery_item` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `item` varchar(255) DEFAULT NULL,
+  `category` enum('PANTRY','FRIDGE','FREEZER','PRODUCE','OTHERS') NOT NULL,
+  `price` decimal(10,2) unsigned NOT NULL,
```

Comment:
For now I would store the price as an int representing the price in pence - there are some arguments to be made for storing them as decimals with more than 2 places to allow the PHP to do the rounding.

A decimal value should round the number (rather than truncate it) but for the first pass at this I'd go with an int, just to get you up and running

Reply:
ok